setup.cfg

```diff
@@ -5,4 +5,3 @@ per-file-ignores =
 
 [tool:pytest]
 flake8-ignore=W605 W503 W291 E203 E501 F821
-addopts = --flake8 --black --mypy
```

Comment:
Was this giving you trouble? Curious as to why you removed the pytest integration. This makes it a bit more annoying to test locally I think as you now have to run these commands all separately, right?

Reply:
Strong +1 on this comment

Reply:
I guess I'd love input on this:

The key reason I removed it is that the plugins to add them to pytest are very out of date, and depend on the typed-ast package. The typed-ast package itself has been discontinued since it has been incorporated into the standard library ast package. typed-ast itself is old enough that there is no binary wheel for M1 laptops even on the newest version. Therefore, if we want to depend on the plugins, M1 laptop users will have to compile typed-ast and this was failing.

I can take another look to see if there's a better solution for this.

Reply:
Actually I think I can keep the plugin.

It turns out the only underlying issue was that it didn't have a high enough requirement to use new black. change incomming.

Reply:
Ok this is fixed.